src/configure

```diff
@@ -818,6 +819,15 @@ do
   --use-cuda=no)
     use_cuda=false;
     shift ;;
+  --with-cudafeat)
```

Comment:
We should perhaps call this --with-cuda-decoding since that is the main reason people would want to use that part of the codebase.. might be less confusing.